src/main/bash/sdkman-init.sh

```diff
@@ -148,3 +148,5 @@ if [[ "$sdkman_auto_env" == "true" ]]; then
 
 	sdkman_auto_env
 fi
+
+[[ "$bash_shell" == "true" ]] && source sdkman_completion.sh
```

Comment:
Have you thought about where we will place this completion script? Will it also live in the `bin` folder, or somewhere else like a `contrib` folder. Either way, we would need to ensure that the `sdkman-completion.sh` is extracted into the right location during the _install.sh_ and _selfupdate.sh_ in the sdkman-hooks app.

Reply:
`contrib` would be a new folder? I've just put it into `bin` to make it work locally 😄 . But I can see why `bin` would not be a good place for it.

Reply:
After sleeping on it, I think it would be fine in the `bin` folder, but perhaps explicitly named as `bash-completion.sh`. That would make things a lot simpler. wdyt?

Reply:
In that case, is it enough to just source `bash-completion.sh`?